### PR TITLE
opera: update to 46.0.2597.26

### PR DESCRIPTION
**Pull request by cr6gtx**
Support for 32 bits Linux has been dropped.

### CODE REVIEW

**Comment by the-maldridge** [comment]
:cry:

File: srcpkgs/opera/template [hunk 1]
```diff
@@ -11,30 +11,18 @@ license="Proprietary"
...
-
-if [ "${XBPS_TARGET_MACHINE}" = "x86_64" ];then
-	_debarch=amd64
-	checksum=2a06172140d22e8740807762f5320007eb6797ed49e8ec6f366ac8a0817caa66
-else
-	_debarch=i386
-	checksum=0fd8497f914efc47eb1ef1c7ce052487e3ca0c4c5474f01ad7e721f6ea08fea8
-fi
-
-distfiles="http://get.geo.opera.com/pub/opera/desktop/${version}/linux/opera-stable_${version}_${_debarch}.deb"
```
Comment (the-maldridge) [inline srcpkgs/opera/template]: please switch back to using the get.geo.opera.com URL, that points into the opera distribution system to ensure you hit a server that will exist, whereas download2 is a regional server.

**Comment by cr6gtx** [comment]
@the-maldridge OK, but this version is already outdated => I should make a new PR.

**Comment by cr6gtx** [comment]
Closed by [#6872](https://github.com/voidlinux/void-packages/pull/6872)

**Pull request closed without merging**